api/app.py

```diff
@@ -44,10 +51,15 @@ def get_all_cases():
 
 
 @app.route('/cases/<case_no>', methods=['GET'])
-def get_case_by_case_number(case_no):
-    """API that returns information about a specific case"""
+def get_case_by_case_number(case_no: str) -> tuple:
+    """API that returns information about a specific case."""
+
+    try:
```

Comment:
Great work catching potential errors when a connection cannot be made with the database.